contributing-guides/maintainers-guide.md

```diff
@@ -58,20 +58,18 @@ as a guideline for current and future maintainers.
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
   (2) have the **review comments addressed**, and
   (3) get **approved reviews by two maintainers**
-  (the second maintainer can perform the merge immediately after accepting.)
+  (4) are at least **24 hours** old unless changes are trivial
 
 - If a PR fails to get a review from a second maintainer after a few days,
-  the first maintainer should ping others for review. If it still lingers around
-  for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
+  the first maintainer should ping others for review.
```

Comment:
This should stay as-is.

Reply:
PRs need 2 approvals so it is wrong.

Reply:
@CleanMachine1 This clause gives the power to org owners to force merge skipping that PR check.

Reply:
But should we (you) even do that? We've never had a PR which had to be merged that urgently but didn't have 2 approvals.

Reply:
So what should we do about this? Leave it?

Reply:
Maybe it can be phrased to show that only in occasions such as bug testing and trivial changes would be acceptable

Reply:
I think even trivial changes should be approved by at least two maintainers. The rule isn't really relevant, so I think we can just leave it.

Reply:
@sbrl could we get your reply about this subject.

Reply:
> But should we (you) even do that? We've never had a PR which had to be merged that urgently but didn't have 2 approvals.

Urgently? No, not really - usually posting urgent urgent urgent ones in Gitter is enough. There have been plenty of times when I've force-merged for example an obvious typo though. Generally speaking I only force-merge uncontroversial changes in periods of very high activity to avoid too much notification spam.

Reply:
Hm, I never noticed that, but it makes sense. I think we can keep the original, but I don't have a strong opinion here.

Reply:
I would side with the modification, this practice does not concern typical maintainers and we should strive to cut down noise.

Reply:
I'm not sure what you mean by "typical maintainer" here @xeruf. Note that the maintainer's guide should be the definitive guide for *all* maintainers - regardless of experience level.